src/platform/Ameba/Logging.cpp

```diff
@@ -27,6 +27,7 @@
 #include <core/CHIPConfig.h>
 #include <support/logging/Constants.h>
 
+#include "Logging.h"
 #include <stdio.h>
 
 #ifdef LOG_LOCAL_LEVEL
```

Comment:
where is LOG_LOCAL_LEVEL used in this file?

Reply:
Removed